include/tester/ssp/SoftphoneManager.hpp

```diff
@@ -55,15 +55,19 @@ class SoftphoneManager
         registerSoftphones(amount * 2);
         for(int i = 0; i < (amount * 2); i += 2)
         {
-            if(_softphones.at(std::to_string(i + START_URI))->isAccountValid())
+            if(_softphones.at(i)->isAccountValid() && _softphones.at(i + 1)->isAccountValid())
             {
-                _softphones.at(std::to_string(i + START_URI))->call(*_softphones.at(std::to_string(i + 1 + START_URI)));
+                _softphones.at(i)->call(*_softphones.at(i + 1));
+                pj_thread_sleep(MILLISECONDS_TO_SECONDS/TEST_SLEEP_DURATION);
             }
         }
-        pj_thread_sleep(TEST_SLEEP_DURATION * MILLISECONDS_TO_SECONDS);
+        pj_thread_sleep(10 * MILLISECONDS_TO_SECONDS);
         for(int i = 0; i < (amount * 2); i += 2)
         {
-            _softphones.at(std::to_string(i + START_URI))->hangup();
+            if(_softphones.at(i)->isActive())
```

Comment:
move the `isActive()` check into `hangup` - that way the method would be safe from crashing if the call is inactive.

Reply:
done